### PR TITLE
[pi-ui] Use pi-ui Tabs component in TabbedPage

### LINKED ISSUE #3222: Replace TabbedPage component with pi-ui Tabs
We can replace our current ``layout/TabbedPage.js`` with the Tabs implementation in ``pi-ui`` dependency since this is going to promote a better integration with ``pi-ui`` and something very close already has been tested with the sidebar component.

### PR DESCRIPTION
**Pull request by bgptr**
Closes #3222

This uses pi-ui `Tabs` component in `TabbedPage`.
Requires decred/pi-ui#376. 
~~Until it will be merged, I edited the package.json.~~ It has been merged and I upgraded the pi-ui package

Also added automated tests for the `TabbedPage` component.

### CODE REVIEW

File: app/components/layout/TabbedPage/TabbedPage.jsx [hunk 1]
```diff
@@ -54,31 +72,35 @@ const TabbedPage = ({
...
-                  // before send, otherwise they will be undfined.
-                  h(content, { ...props }, null);
-              return (
-                <Tab label={label} key={path}>
-                  <div key={`${path}-key`}>{element}</div>
-                </Tab>
-              );
-            })}
-        </Tabs>
+        <ThemeProvider themes={customThemes} defaultThemeName={themeName}>
```
Comment (tiagoalvesdulce) [inline app/components/layout/TabbedPage/TabbedPage.jsx:75]: 👍

Comment (tiagoalvesdulce) [inline app/components/layout/TabbedPage/TabbedPage.jsx:75]: 👍

File: app/components/layout/TabbedPage/TabbedPage.jsx [hunk 2]
```diff
@@ -39,12 +46,23 @@ const TabbedPage = ({
...
-  themes.tabDefaultBackgroundColor = themes.tabDefaultBackgroundColor
-    ? themes.tabDefaultBackgroundColor
-    : "tab-default-background-tabbedpage";
-  themes.tabDefaultBorderColor = themes.tabDefaultBorderColor
-    ? themes.tabDefaultBorderColor
-    : "tab-default-border-tabbedpage";
+  const { theme } = useTheme();
+  const themeName = "tabbedPageCustomThemeName";
+
+  const customThemes = {
```
Comment (tiagoalvesdulce) [inline app/components/layout/TabbedPage/TabbedPage.jsx:52]: 👍

**Review by tiagoalvesdulce — COMMENTED** [review]
You did a great job using nested ThemeProviders! 👏

**Review by matheusd — APPROVED** [review]

**Pull request merged**